common.sh

```diff
@@ -19,34 +19,34 @@ function setup_docker_machine() {
 
         # Check if the docker machine exists already, create one if not.
         if [[ $? == 1 ]]; then
-            notice "No ${MACHINE} environment found"
+            echo "No ${MACHINE} environment found"
```

Comment:
Why pull in another dependency? What value does it provide? I couldn't find it on a quick googles.